Library/Formula/rust.rb

```diff
@@ -44,7 +44,6 @@ def install
     args = ["--prefix=#{prefix}"]
     args << "--disable-rpath" if build.head?
     args << "--enable-clang" if ENV.compiler == :clang
-    args << "--release-channel=beta" unless build.head?
```

Comment:
From reading the current [configure](https://github.com/rust-lang/rust/blob/e5394240a295650b567aa406b4a0e1e3a6749a5f/configure#L645-L656) and (make.mk)[https://github.com/rust-lang/rust/blob/e5394240a295650b567aa406b4a0e1e3a6749a5f/mk/main.mk#L27-L54] scripts, it looks like `nightly` is correct for HEAD, and we need `stable` when building 1.0